addons/binding/org.openhab.binding.osramlightify/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="osramlightify" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="color">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>Controls the color and brightness of the light. Can also be used to switch the light on and off.</description>
+		<category>Color</category>
+	</channel-type>
+
+	<channel-type id="dimmer">
+		<item-type>Dimmer</item-type>
+		<label>Brightness</label>
+		<description>Controls the brightness of the light. Can also be used to switch the light on and off.</description>
+		<category>DimmableLight</category>
+		<state min="0" max="100" pattern="%d %%"/>
+	</channel-type>
+
+	<!-- Everyone uses dimmers for white temperature so we have to have one too -->
+	<channel-type id="temperature">
```

Comment:
Hue uses color_temperature but LIFX uses temperature. This is especially ironic since Hue is modal. It can be either colour _or_ white and temperature only affects white mode. So that is *not* colour temperature at all. LIFX on the other hand is not modal and temperature can be varied while colour is fully saturated (although I suspect it only alters the temperature of the white LEDs and doesn't actually adjust the mapping of the colour ones). So there temperature _could_ be considered to be colour temperature.

OSRAM Lightify lights are modal so the temperature channel is *not* colour temperature, just white temperature.